### ISSUE TITLE
Accessibility - 2.4.7 Focus visible

### ISSUE THREAD

**Issue by phucduong86** [shiffman/shiffman.net#89]
Results for: (https://shiffman.net)
Findings: Outline styling removed on GET UPDATES button. Recommend alternative styling for focus state.

All pages:
Keyboard user cannot see when a navigation link is focused because it is positioned off screen.

**Comment by phucduong86** [comment]
First issue resolved - PR #96

**Comment by phucduong86** [comment]
Resolved by PR #98
